lib/actors/teacher_timetable_import.rb

```diff
@@ -0,0 +1,53 @@
+require 'celluloid'
+require 'actors/teacher_source'
+require 'actors/timetable_source'
+require 'actors/timetable_transformer'
+require 'actors/event_destination'
+require 'actors/teacher_timetables_cleaner'
+
+# An actor pipeline system for importing teacher timetables from KOSapi.
+#
+# The inputs are: faculty semester, teacher usernames for that semester,
+# semester periods (all from the DB) and teacher timetables (fetched from KOSapi).
+#
+# The output is events of type 'teacher_timetable' that are written to the DB.
+#
+class TeacherTimetableImport
```

Comment:
I am in favor of TeacherTimetableSlot, because it is closest how the thing is represented in KOS(api). Also we apparently used TeacherTimetableSlot naming in KOSapiClient.